### PR TITLE
Update docstring of the `heatmaps_to_keypoints` 

### LINKED ISSUE #7148: docstring for `heatmaps_to_keypoints` does not match return values.
The docstring for the function `heatmaps_to_keypoints` in `torchvision/models/detection/roi_heads.py` does not match the return values.
```python
def heatmaps_to_keypoints(maps, rois):
    """Extract predicted keypoint locations from heatmaps. Output has shape
    (#rois, 4, #keypoints) with the 4 rows corresponding to (x, y, logit, prob)
    for each keypoint.
    """
...
return xy_preds.permute(0, 2, 1), end_scores
```
The function returns 2 tensors.  The first tensor has the shape `(#rois, #keypoints, 3)` with the last dimension corresponding to `(x, y, vis)`.  The second tensor has shape `(#rois, #keypoints)` the values of which correspond to... logits?  Some explanation on what these `end_scores` are used for and/or how to use them would also be helpful.

### PR DESCRIPTION
**Pull request by rajveer43**
close #7148

### CODE REVIEW

**Review by NicolasHug — COMMENTED** [review]
Thanks for the PR @rajveer43 . If I may ask, how did you come up with this docstring? It looks quite verbose with a few repeated sections.
